scipy/optimize/_linprog.py

```diff
@@ -142,13 +142,13 @@ def linprog_terse_callback(res):
             feasible solution is sought and the T has an additional row
             representing an alternate objective function.
         status : int
-            An integer representing the exit status of the optimization::
+            An integer representing the exit status of the optimization:
```

Comment:
This is another thing that shows up occasionally - rendering return code lists as literal blocks. One could argue that it's supposed to be that way, but I think it's more important for it to be consistent.

Reply:
Literal blocks are easier to align on the screen, otherwise you get wonky lists which look quite ugly.

Reply:
> Literal blocks are easier to align on the screen, otherwise you get wonky lists which look quite ugly.

I agree that the lists go too far to the left, but isn't that more of a problem with the Sphinx theme? I think that's something we can fix across the board with a change in the right place.

> This change isn't being displayed correctly.
Thanks. It was supposed to be more like:
![image](https://user-images.githubusercontent.com/6570539/99911743-498ded80-2cab-11eb-85e7-ece3464894ea.png)

I'll fix.

Reply:
Looks like I did fix this.